angularjs.html.markdown

```diff
@@ -0,0 +1,709 @@
+---
+language: AngularJS/JavaScript
+contributors:
+    - ["Walter Cordero", "http://waltercordero.com"]
+translators:
+    - ["Walter Cordero", "http://waltercordero.com"]     
+filename: learnangular.html
+---
+
+Feedback is welcome! You can find me in:
```

Comment:
@adambard should we cleanup tutorials from this? We already have the contributor link at the top